src/server/jobs-ete/infra/repositories/apiPoleEmploiJobEte.repository.ts

```diff
@@ -40,12 +39,18 @@ export class ApiPoleEmploiJobEteRepository implements OffreRepository {
 	async get(id: OffreId): Promise<Either<Offre>> {
 		try {
 			const response = await this.httpClientServiceWithAuthentification.get<OffreResponse>(`/${id}`);
-			if (response.status === 204) {
-				return createFailure(ErreurMétier.CONTENU_INDISPONIBLE);
+			if (this.apiPoleEmploiOffreErrorManagementGet.isError(response)) {
+				return this.apiPoleEmploiOffreErrorManagementGet.handleFailureError(response, {
+					apiSource: 'API Pole Emploi',
+					contexte: 'détail job d‘été', message: '[API Pole Emploi] impossible de récupérer un job d‘été',
+				});
 			}
 			return createSuccess(mapOffre(response.data));
-		} catch (e) {
-			return handleGetFailureError(e, 'job ete', this.loggerService);
+		} catch (error) {
+			return this.apiPoleEmploiOffreErrorManagementGet.handleFailureError(error, {
+				apiSource: 'API Pole Emploi',
+				contexte: 'détail job d‘été', message: '[API Pole Emploi] impossible de récupérer un job d‘été',
```

Comment:
```suggestion
				contexte: 'détail job d‘été', message: '[API Pole Emploi] impossible de récupérer le détail d'un job d‘été',
```
pareill pour le test du dessus